.github/workflows/test.yaml

```diff
@@ -18,18 +18,18 @@ jobs:
         with:
           go-version: "1.20"
 
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
         with:
           path: ~/go/pkg/mod
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: ${{ runner.os }}-go-
```

Comment:
This one is not necessary any more. Latest releases of the go action already perform module caching. Just use `actions/setup-go@v5` in the "Setup Go action".

Reply:
done at  https://github.com/3scale-ops/basereconciler/pull/17/commits/b584b134f3b479ed4b8fed1eccc99052b80fe08d